### PR TITLE
Fix table width

### PR DESCRIPTION
**Pull request by thecalcc**
SDESK-7116

I didn't find the ticket specific to this issue but we can consider it a part of the general fixes for multi-line quote

### CODE REVIEW

**Comment by tomaskikutis** [comment]
actual tables shouldn't be set to 100% by default. Add a config option so it's only applied to pull quote

**Comment by thecalcc** [comment]
> actual tables shouldn't be set to 100% by default. Add a config option so it's only applied to pull quote

Actually we don't need a config option since we already have className which we can check against whether it's a multi line quote or not

**Comment by tomaskikutis** [comment]
Classnames are for styling and you shouldn't be using them for anything else. If I could, I would write a lint rule not allowing to compare a classname using `===` :D

Use a separate prop like `fullWidth: boolean` and keep classname decoupled.

File: scripts/core/editor3/components/tests/tables.spec.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ describe('editor3.component.table-cell', () => {
     it('should render', () => {
         const wrapper = shallow(
             <TableCell
+                fullWidth={{}}
```
Comment (tomaskikutis) [inline scripts/core/editor3/components/tests/tables.spec.tsx:56]: how are you passing an empty object where a boolean is required?

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**